### PR TITLE
Child care option

### PR DESCRIPTION
**Pull request by plaindocs**
Basic. Any improvements to suggest?

### CODE REVIEW

File: docs/conf/eu/2017/sponsorship/prospectus.rst [hunk 1]
```diff
@@ -189,6 +189,12 @@ Sponsorship of the official party includes:
 - Mention of company on schedule for the party, and an on-stage thanks
 - Your logo on a banner or roll up at the party venue (provided by sponsor)
 
+Child Care
+-----------
+
+Many of our attendees have children, sponsoring day care makes it  easier for
```
Comment (thatdocslady) [inline docs/conf/eu/2017/sponsorship/prospectus.rst:195]: double-space action ;)

also, do we not put a price on this?

Comment (plaindocs) [reply docs/conf/eu/2017/sponsorship/prospectus.rst:195]: We'd have to spec it out if we wanted to put a price on it no?

Comment (thatdocslady) [reply docs/conf/eu/2017/sponsorship/prospectus.rst:195]: True, but maybe we can add more details like "exact sponsorship amount TBD based on demand, but would probably be in the hundreds" or something more elegant like that ;)

Comment (plaindocs) [reply docs/conf/eu/2017/sponsorship/prospectus.rst:195]: I'm all about the elegant.

**Pull request merged**